scripts/deploy-incentivisedvotinglockup.js

```diff
@@ -0,0 +1,58 @@
+const hre = require("hardhat");
+const { VOTINGLOCKUP_PARAMS, AIRDROP_PARAMS } = require("../params");
+
+async function main() {
+  const [deployer] = await hre.ethers.getSigners();
+  const network = hre.network.name;
+
+  // We get the contract to deploy
+  const IncentivisedVotingLockup = await hre.ethers.getContractFactory(
+    "IncentivisedVotingLockup",
+    deployer
+  );
+
+  const stakingToken =
+    network === "kovan"
+      ? "0x567e482AF973187648Af9FC56d2Caec212c1CAca"
+      : AIRDROP_PARAMS.TOKEN_ADDRESS;
+
+  const owner =
+    network === "kovan" ? deployer.address : VOTINGLOCKUP_PARAMS.OWNER;
```

Comment:
Is it possible to lookup / check via `chainId` instead?  
Something like `VOTINGLOCKUP_PARAMS.OWNER[chainId]`